mocha.css

```
@@ -334,20 +334,47 @@ body {
   z-index: 1;
 }
 
-#mocha-stats .progress {
+#mocha-stats .progress-contain {
   float: right;
-  padding-top: 0;
-
-  /**
-   * Set safe initial values, so mochas .progress does not inherit these
-   * properties from Bootstrap .progress (which causes .progress height to
-   * equal line height set in Bootstrap).
-   */
-  height: auto;
-  -webkit-box-shadow: none;
-  -moz-box-shadow: none;
-  box-shadow: none;
-  background-color: initial;
+  padding: 0;
+}
+
+#mocha-stats .progress-element {
+  visibility: hidden;
+  width: 40px;
+  height: 20px;
+  position: absolute;
+  top: 11px; /* padding */
+  display: block;
+}
+
+#mocha-stats .progress-text {
+  text-align: center;
+  position: absolute;
+  width: 40px;
+  display: block;
+  top: 11px; /* padding */
+}
+
+#mocha-stats .progress-ring {
+  width: 40px;
+  height: 40px;
```

[Style] Later down there are comments like `/* half of width */` to indicate the relationships between values. How about using CSS variables to make the relationships explicitly stated in code?

```suggestion
  --ring-size: 40px;
  width: var(--ring-size);
  height: var(--ring-size);
```

...and later down, like this:

```css
  cx(calc(var(--ring-size) / 2);
```

Come to think of it, I bet `--ring-size` could actually be put in the `#mocha-stats` so that `.progress-element`, `progress-text`, etc. can use it.